src/us/kbase/common/executionengine/CallbackServer.java

```diff
@@ -1,10 +1,6 @@
 package us.kbase.common.executionengine;
 
-import java.io.ByteArrayOutputStream;
-import java.io.IOException;
-import java.io.OutputStream;
-import java.io.PrintWriter;
-import java.io.StringWriter;
+import java.io.*;
```

Comment:
This makes it difficult to figure out where a name comes from - explicit imports are more readable imo

Reply:
This is the result of the optimize imports button. Usually I just cmd click a variable in intellij and it tells me which package it came from.

Reply:
Which you don't have when you're doing github reviews or working in a text editor

Reply:
True, good point.  I'm used to cloning repos, loading them up in my IDE and running the tests locally, but it is nice when Travis runs the tests for you.